src/Presentation/Presenters/CommentListJsonPresenter.php

```diff
@@ -8,19 +8,20 @@
 use WMDE\Fundraising\DonationContext\UseCases\ListComments\CommentList;
 
 /**
- * @licence GNU GPL v2+
- * @author Jeroen De Dauw < jeroendedauw@gmail.com >
+ * @license GNU GPL v2+
  */
 class CommentListJsonPresenter {
 
 	public function present( CommentList $commentList ): array {
+		// TODO Translate keys into English once old skins are phased out
 		return array_map(
 			function( CommentWithAmount $comment ) {
 				return [
 					'betrag' => $comment->getDonationAmount(),
 					'spender' => $comment->getAuthorName(),
 					'kommentar' => $comment->getCommentText(),
 					'datum' => $comment->getDonationTime()->format( 'r' ),
+					'datum_formatted' => $comment->getDonationTime()->format( 'd.m.Y \u\m H:i:s \U\h\r' ),
```

Comment:
Renders as `15.08.2019 um 16:38:30 Uhr` because formatting that in JavaScript would be SO MUCH more effort than doing it in PHP straight away. `'datum' => $comment->getDonationTime()->format( 'r' ),` is truly a pain to format in JavaScript, let's not waste time on that.

For backwards compatibility reasons I have left the existing `datum` field in place. I assume that you will not like the hard-coded German string (_"01.02.2003 um 12:34:56 Uhr"_) here but I personally don't think it's worth putting effort into creating a better solution for such a low-visibility page. If you still want to make it localizable, go ahead. Again though, I would really say let's not spend too much time on this tiny bit and maybe refactor this after the old skins are gone.